routes/messages/commands/get_started.js

```diff
@@ -36,42 +36,80 @@ async function getMultipleBooks(event) {
 
   const text = userCategories.length === 0 ? firstTimeText : introText;
 
-  const buttons = [
+  // const buttons = [
+  //   {
+  //     type: 'postback',
+  //     title: 'Browse Books',
+  //     payload: JSON.stringify({
+  //       command: 'browse'
+  //     })
+  //   }
+  // ];
+
+  // if (userLibraries.length) {
+  //   buttons.push({
+  //     type: 'postback',
+  //     title: 'View Library',
+  //     payload: JSON.stringify({ command: 'library' })
+  //   });
+  // }
+
+  // buttons.push({
+  //   type: 'postback',
+  //   title: 'Get Help',
+  //   payload: JSON.stringify({ command: 'help' })
+  // });
+
+  // return [
+  //   {
+  //     attachment: {
+  //       type: 'template',
+  //       payload: {
+  //         template_type: 'button',
+  //         text,
+  //         buttons
+  //       }
+  //     }
+  //   }
+  // ];
```

Comment:
We can just delete these.  It's Git so if we need them, we'll go back to previous versions